clients/web/static/girder-swagger.js

```diff
@@ -1,6 +1,11 @@
 $(function () {
+    var apiRoot = $('#g-global-info-apiroot').text().replace(
+        '%HOST%', 'http://' + window.location.host);
```

Comment:
I assume "window.location.protocol" could replace the hard-coded http://?

roni

On Fri, Apr 3, 2015 at 10:26 AM, David Manthey <notifications@github.com>
wrote:

> In clients/web/static/girder-swagger.js
> <https://github.com/girder/girder/pull/788#discussion_r27731755>:
>
> > @@ -1,6 +1,11 @@
> >  $(function () {
> > +    var apiRoot = $('#g-global-info-apiroot').text().replace(
> > +        '%HOST%', 'http://' + window.location.host);
>
> This replace only occurs in tests, not in actual deployment. We don't
> currently test the swagger docs (but we should -- see issue #320
> <https://github.com/girder/girder/issues/320>), so this doesn't matter
> yet.
> … <#14c7fad53ae8cd51_>
> On Fri, Apr 3, 2015 at 10:25 AM, Patrick Reynolds <
> notifications@github.com> wrote: In clients/web/static/girder-swagger.js <
> https://github.com/girder/girder/pull/788#discussion_r27731665>: > @@
> -1,6 +1,11 @@ > $(function () { > + var apiRoot =
> $('#g-global-info-apiroot').text().replace( > + '%HOST%', 'http://' +
> window.location.host); What happens if we're https only? — Reply to this
> email directly or view it on GitHub <
> https://github.com/girder/girder/pull/788/files#r27731665>.
>  -- David Manthey R&D Engineer Kitware Inc. (518) 881-4439
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/girder/girder/pull/788/files#r27731755>.
>